pkg/driver/controller_utils.go

```diff
@@ -150,6 +150,13 @@ func (cs *controller) validateVolumeCreateReq(req *csi.CreateVolumeRequest) erro
 				)
 			}
 		}
+		if mode := volcap.GetAccessMode(); mode != nil {
+			modeName := csi.VolumeCapability_AccessMode_Mode_name[int32(mode.GetMode())]
+			// we only support SINGLE_NODE_WRITER
+			if mode.GetMode() != csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER {
+				return status.Errorf(codes.InvalidArgument, "unsupported access mode: %s", modeName)
```

Comment:
How about having error message like the following:
```go
return status.Errorf(codes.InvalidArgument, "invalid volume capabilities requested(%s). Only SINGLE_NODE_WRITER is supported ('accessModes.ReadWriteOnce' on Kubernetes)", modeName)
```

Reply:
above was intentional to make it more readable, IMO error clears that its not supported version, and we don't need to spcifically mention about the capabilites (that's internal for the driver)

Reply:
I am thinking in lines of lame users who don't understand the error so thought of specifying readwriteonce.

Reply:
Here unsupported mode can be anything other then ReadWriteMany/ReadOnly/  etc, so we can't just mentioned  RWM here as an error.

To make you happy,, i just change it to
```go
 status.Errorf(codes.InvalidArgument,"only SINGLE_NODE_WRITER supported, unsupported access mode requested: %s",modeName)
```